### DISCUSSION TITLE
タイトルに半角スペースがあると OGP 画像のタイトルが奇妙な位置で改行されることがある

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#891] [要望・アイデア・不具合]
タイトルに半角スペースがある場合、OGP 画像内のタイトルが奇妙な位置で改行されます。

例えば、「wrapstuff で図表に文章を回り込みさせたい (https://qiita.com/Yarakashi_Kikohshi/items/4922deb4038160bf796d )」では、wrapstuff の位置で改行されています。（[当該 OGP][ogp]）

[ogp]: https://qiita-user-contents.imgix.net/https%3A%2F%2Fcdn.qiita.com%2Fassets%2Fpublic%2Farticle-ogp-background-412672c5f0600ab9a64263b751f1bc81.png?ixlib=rb-4.0.0&w=1200&mark64=aHR0cHM6Ly9xaWl0YS11c2VyLWNvbnRlbnRzLmltZ2l4Lm5ldC9-dGV4dD9peGxpYj1yYi00LjAuMCZ3PTk3MiZoPTM3OCZ0eHQ9d3JhcHN0dWZmJTIwJUUzJTgxJUE3JUU1JTlCJUIzJUU4JUExJUE4JUUzJTgxJUFCJUU2JTk2JTg3JUU3JUFCJUEwJUUzJTgyJTkyJUU1JTlCJTlFJUUzJTgyJThBJUU4JUJFJUJDJUUzJTgxJUJGJUUzJTgxJTk1JUUzJTgxJTlCJUUzJTgxJTlGJUUzJTgxJTg0JnR4dC1hbGlnbj1sZWZ0JTJDdG9wJnR4dC1jb2xvcj0lMjMyMTIxMjEmdHh0LWZvbnQ9SGlyYWdpbm8lMjBTYW5zJTIwVzYmdHh0LXNpemU9NTYmcz1hYjA3OGZlYWM0ZWRiMzQ1MWM5NGIwZGQzMjc0NTMwYw&mark-x=142&mark-y=57&blend64=aHR0cHM6Ly9xaWl0YS11c2VyLWNvbnRlbnRzLmltZ2l4Lm5ldC9-dGV4dD9peGxpYj1yYi00LjAuMCZoPTc2Jnc9NzcwJnR4dD0lNDBZYXJha2FzaGlfS2lrb2hzaGkmdHh0LWNvbG9yPSUyMzIxMjEyMSZ0eHQtZm9udD1IaXJhZ2lubyUyMFNhbnMlMjBXNiZ0eHQtc2l6ZT0zNiZ0eHQtYWxpZ249bGVmdCUyQ3RvcCZzPTM2MzRlYjQ2MzU0MzkxNzFhYjk4OGI2YjZkNGYyZjc4&blend-x=142&blend-y=486&blend-mode=normal&s=0af02919eb834e1a09c98ecb8a0a85b4

このような表示はあまり意図していないため、できれば改行されないように表示されていてほしいです。

**Comment by kimkim0814** [comment]
@Yarakashi-Kikohshi
ご不便をおかけして申し訳ございません。

imgixの機能で改行しているのですが、タイトルが一定の文字数以上になると最後の半角スペースの位置で改行されるようです。

現時点では、タイトル内の半角スペースを削除することで改行を回避できます。
お手数をおかけいたしますが、タイトルを調整していただけますと幸いです。

**Reply by yarakos95** [reply]
@kimkim0814 様

パターンが見えている分こちら側で対応できるのはありがたいのですが、ラテン文字と和文の間にスペースを設けるのが癖付いているために面倒さを感じてしまっています。
また、記事を投稿した際に OGP 画像が見えないため、タイトルには半角スペースを入れないことを忘れそうだなとも思っています。

そこで提案なのですが、以下のいずれかのような対応していただくことは可能でしょうか。

- タイトルに半角スペースが含まれていると警告を出す
- 「記事の公開設定」画面で OGP 画像を表示する

記事を投稿する際に OGP 画像にも気を配れるような環境であるとより良いように思います。

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
ご提案いただきありがとうございます！

> タイトルに半角スペースが含まれていると警告を出す

こちらは、警告が出るようになると嫌な方もいそうなため、避けたいと考えています🙏
我々としては、半角スペースがあっても改行されないようにしたいと考えており、一度 imgixに問合せを行う予定です。

> 「記事の公開設定」画面で OGP 画像を表示する

 imgixの回答内容を踏まえて、検討したいと思います！

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi 
お待たせしました！
imgix のサポートに対応いただき、半角スペースで改行されないように修正いただきました✨

[【OGP画像】](https://qiita-user-contents.imgix.net/https%3A%2F%2Fcdn.qiita.com%2Fassets%2Fpublic%2Farticle-ogp-background-412672c5f0600ab9a64263b751f1bc81.png?ixlib=rb-4.0.0&w=1200&mark64=aHR0cHM6Ly9xaWl0YS11c2VyLWNvbnRlbnRzLmltZ2l4Lm5ldC9-dGV4dD9peGxpYj1yYi00LjAuMCZ3PTk3MiZoPTM3OCZ0eHQ9d3JhcHN0dWZmJTIwJUUzJTgxJUE3JUU1JTlCJUIzJUU4JUExJUE4JUUzJTgxJUFCJUU2JTk2JTg3JUU3JUFCJUEwJUUzJTgyJTkyJUU1JTlCJTlFJUUzJTgyJThBJUU4JUJFJUJDJUUzJTgxJUJGJUUzJTgxJTk1JUUzJTgxJTlCJUUzJTgxJTlGJUUzJTgxJTg0JnR4dC1hbGlnbj1sZWZ0JTJDdG9wJnR4dC1jb2xvcj0lMjMyMTIxMjEmdHh0LWZvbnQ9SGlyYWdpbm8lMjBTYW5zJTIwVzYmdHh0LXNpemU9NTYmcz1hYjA3OGZlYWM0ZWRiMzQ1MWM5NGIwZGQzMjc0NTMwYw&mark-x=142&mark-y=57&blend64=aHR0cHM6Ly9xaWl0YS11c2VyLWNvbnRlbnRzLmltZ2l4Lm5ldC9-dGV4dD9peGxpYj1yYi00LjAuMCZoPTc2Jnc9NzcwJnR4dD0lNDBZYXJha2FzaGlfS2lrb2hzaGkmdHh0LWNvbG9yPSUyMzIxMjEyMSZ0eHQtZm9udD1IaXJhZ2lubyUyMFNhbnMlMjBXNiZ0eHQtc2l6ZT0zNiZ0eHQtYWxpZ249bGVmdCUyQ3RvcCZzPTM2MzRlYjQ2MzU0MzkxNzFhYjk4OGI2YjZkNGYyZjc4&blend-x=142&blend-y=486&blend-mode=normal&s=0af02919eb834e1a09c98ecb8a0a85b4)

**Reply by yarakos95** [reply]
改行が解消されていることを確認しました
修正いただきありがとうございます 🎉
